src/utils/validation.js

```diff
@@ -26,15 +33,25 @@ const createArticleValidator = ({ body }, res, next) => {
     });
   }
 
+  if (body.type === 'text' && body.videoUrl) {
+    errors.video = 'errors.forbiddenForTypeText';
+  }
+
   return next(!isEmpty(errors) && new ValidationError(errors));
 };
 
 const updateArticleValidator = ({ body }, res, next) => {
   const errors = {};
 
+  ['brand', 'collection'].forEach(field => {
```

Comment:
please extract it in function & remove copy-pasting above

Reply:
done.